client/state/sharing/publicize/test/reducer.js

```diff
@@ -25,30 +29,36 @@ import { useSandbox } from 'test/helpers/use-sinon';
 describe( 'reducer', () => {
 	describe( '#fetchingConnections()', () => {
 		it( 'should set fetching to true for fetching action', () => {
-			const state = fetchingConnections( null, {
-				type: PUBLICIZE_CONNECTIONS_REQUEST,
-				siteId: 2916284
-			} );
+			[ PUBLICIZE_CONNECTION_REQUEST, PUBLICIZE_CONNECTIONS_REQUEST ].map( ( type ) => {
```

Comment:
Looking at the reducer tests, the current expectation is that connections missing from an response should be deleted. Fetching a single connection feels closer related to create or update as it should only care about that specific one. See https://github.com/Automattic/wp-calypso/pull/9273/files#diff-ee58ef3f817ca561d7c6eef1ee213b44R70

Maybe I should model it more like those?